typescript-test-samples/launchdarkly-lambda-dynamodb/readme.md

```diff
@@ -0,0 +1,90 @@
+# Using LaunchDarkly for Testing in AWS Lambdas
+
+## Introduction
+
+The goal of this project is to help you get started using LaunchDarkly feature flags to test new code and features within a serverless application using AWS Lambda. The project includes everything you'll need to get started integrating feature flags into your Lambdas using the Node.js runtime.
+
+The project uses the [AWS Cloud Development Kit](https://aws.amazon.com/cdk/) (CDK) for deployment. 
+
+---
+
+## Contents
+
+- [Introduction](#introduction)
+- [Contents](#contents)
+- [About this Example](#about-this-example)
+- [The LaunchDarkly SDKs](#the-launchDarkly-sdks)
+- [Deploying via the AWS CDK](#deploying-via-the-aws-cdk)
+
+---
+
+## Contents
```

Comment:
Incorrect heading... or maybe not? This section seems to belong after "About this Example", maybe as a subheading in there? It seems to assume I understand the "About this Example" section.